tests/impl.cpp

```diff
@@ -3429,7 +3429,30 @@ result_t test_mm_srli_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_slli_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    result_t result1, result2;
+    const int64_t *_a = (const int64_t *) impl.mTestIntPointer1;
+    int count = 3; /* imm8 <= 63 */
+
+    int64_t d0 = _a[0] << count;
+    int64_t d1 = _a[1] << count;
+
+    __m128i a = do_mm_load_ps((const int32_t *) _a);
+    __m128i c = _mm_slli_epi64(a, count);
+    result1 = validateInt64(c, d0, d1);
```

Comment:
If `result1` was wrong, then we can return `TEST_FAIL`.

Reply:
Actually I met a problem. What is the expected result when count is greater than 63?

Reply:
Quote Intel Intrinsic Guide:
```
FOR j := 0 to 1
	i := j*64
	IF imm8[7:0] > 63
		dst[i+63:i] := 0
	ELSE
		dst[i+63:i] := ZeroExtend64(a[i+63:i] << imm8[7:0])
	FI
ENDFOR
```

You shall check the lowest byte (i.e. `count & 0xFF`)

Reply:
You can check the value of `result1`. If it is false, then return `TEST_FAIL`. Thus, we don't need `if ((result1 == TEST_SUCCESS) && (result2 == TEST_SUCCESS))` statement anymore.